panels/user-accounts/data/user-accounts-dialog.ui

```diff
@@ -106,329 +106,336 @@
               </packing>
             </child>
             <child>
-              <object class="GtkGrid" id="user-options">
+              <object class="GtkScrolledWindow">
```

Comment:
Do you also want to remove the scrolled window from app-permissions.ui? It might feel a little weird to have nested scrollable areas?

Reply:
The one for ‘Restrict Apps’? No, I think that would make the dialogue very long and look quite weird.